starlette/applications.py

```diff
@@ -107,7 +107,14 @@ def debug(self) -> bool:
         return self._debug
 
     @debug.setter
-    def debug(self, value: bool) -> None:
+    def debug(self, value: bool) -> None:  # pragma: no cover
+        """
+        We no longer document this API, and its usage is discouraged.
+        Instead you should use the following approach:
+
+        app = Starlette(debug=True)
+        """
+
```

Comment:
```suggestion
    def debug(self, value: bool) -> None:
```

Reply:
Reverted this change for later discussion.